sentry-core/src/main/java/io/sentry/core/MainEventProcessor.java

```diff
@@ -20,6 +20,7 @@ public SentryEvent process(SentryEvent event) {
       event.setThreads(sentryThreadFactory.getCurrentThreads());
     }
 
+    // DefaultAndroidEventProcessor also sets it, which one is correct or which one should be 1st?
```

Comment:
@bruno-garcia

Reply:
`DefaultAndroidEventProcessor` should be added second and run after this one.
But can't `sentry-android` just set the value to `SentryOptions` before passing it to `Sentry.init`?

Reply:
MainEventProcessor always run 1st so that's fine.